Brokerages/WebSocketWrapper.cs

```diff
@@ -41,13 +42,17 @@ public void Initialize(string url)
             _url = url;
             _wrapped = new WebSocket(url)
             {
-                Log = { Output = (data, file) => { Log.Trace(data.Message); } }
+                Log =
+                {
+                    Level = Config.GetBool("websocket-log-trace") ? LogLevel.Trace : LogLevel.Error,
+                    Output = (data, file) => { Log.Trace(WhoCalledMe.GetMethodName(3) + "(): " + data.Message, true); }
```

Comment:
Nice usage of `WhoCalledMe` 👍

Reply:
Consider using string interpolation here instead of concatenation

Reply:
Updated and also added a comment to clarify the origin of the magic number `3` which matches the usage pattern of the [Logger](https://github.com/sta/websocket-sharp/blob/master/websocket-sharp/Logger.cs) class in the `WebSocketSharp` library.